core/storage/user/gae_models.py

```diff
@@ -683,23 +681,22 @@ def get(cls, model_id):
             model_id: str. The model id of the user and the skill.
 
         Returns:
-            list. All skill mastery models where user_id belongs to the
-                entered user are returned as a list where each element
-                is one skill mastery model.
+            dict. Skill mastery model with id equal to model_id.
         """
-        return cls.query(cls.id == model_id).fetch()
+        return cls.get_by_id(model_id)
 
     @classmethod
     def get_multi(cls, model_ids):
-        """Returns all skill mastery models of a different model ids.
+        """Returns all skill mastery models for the given model ids.
 
         Args:
-            model_ids: list. List containing model ids of users and
+            model_ids: list(str). List containing model ids of users and
                 skills.
 
         Returns:
-            list. All skill mastery models for all model_id in the list
+            list(dict). All skill mastery models for all model_id in the list
```

Comment:
Pretty sure the return values are not dicts.

Again, check the base class. The method there might already suffice, and you might not need to write a replacement.

Reply:
That somewhat surprises me. What's the trace?

If you look at base_model.gae_models.BaseModel.get_multi() then it looks like it's using ndb.get_multi() behind the scenes too. So I don't quite understand why one errors and the other does not.

Reply:
Oh I'm sorry about this. This happens in case of get() requests only. Because in the base class get requests resulting in ```None``` are handled like this: 
```
        if strict and entity is None:
            raise cls.EntityNotFoundError(
                'Entity for class %s with id %s not found' %
                (cls.__name__, entity_id))
        return entity
```

Reply:
Oh I'm sorry about this. This happens in case of get() requests only. Because in the base class get requests resulting in ```None``` are handled like this: 
```
        if strict and entity is None:
            raise cls.EntityNotFoundError(
                'Entity for class %s with id %s not found' %
                (cls.__name__, entity_id))
        return entity
```

Reply:
It's safe to remove get_multi() method from here